src/GovUk.Education.ExploreEducationStatistics.Publisher/Functions/PublishImmediateReleaseContentFunction.cs

```diff
@@ -39,7 +39,7 @@ public class PublishImmediateReleaseContentFunction
         /// <param name="executionContext"></param>
         /// <param name="logger"></param>
         /// <returns></returns>
-        [FunctionName("PublishReleaseContent")]
+        [FunctionName("PublishImmediateReleaseContent")]
```

Comment:
I've just checked and it looks like every time the publisher functions project is deployed it automatically cleans up any unused functions left behind in the old deployment.

We've also renamed functions fairly recently and not seen any problems. This is one of the renames that got missed.

I spotted it here because as I was developing I ran into some exceptions which mentioned this name in the logs and spent quite a long time trying to track down the error before I realised the function name in the logs was wrong.